scripts/codegen.ts

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env zx
+import { convertCase } from '@axelarjs/utils/case-conversion';
+import { capitalize } from '@axelarjs/utils/string';
```

Comment:
Do we need the full utils dep for this? Maybe some lighter dependency for this scenario works instead?

Reply:
This is tree-shakeable and won't affect any client